README.md

```diff
@@ -267,13 +267,13 @@ This way, when emerging from Signed or Anonymous mode, you can be sure you'll ac
 
 ## SameSite cookies
 
-> :warning: **WARNING**: If you see the warnings about `SameSite` cookies in Chrome or Firefox, please upgrade the TreasureData JS SDK to version `2.4.1`
+> :warning: **WARNING**: If you see the warnings about `SameSite` cookies in Chrome or Firefox, please upgrade the TreasureData JS SDK to version `2.4.2`
```

Comment:
May be just write "to version of at least 2.4.1"? So that we don't have to change it every time in  future release?